apiserver/facades/client/usermanager/usermanager_test.go

```diff
@@ -821,3 +821,88 @@ func (s *userManagerSuite) TestRemoveUserBulkSharedModels(c *gc.C) {
 	c.Assert(alice.IsDeleted(), jc.IsTrue)
 
 }
+
+func (s *userManagerSuite) TestResetPassword(c *gc.C) {
```

Comment:
I feel like this is more of a "featuretest" as this will be tested as part of add-user through CLI and reset their password - this will create a user without password.
This will be in addition to an existing, similar state-layer test.